test/input_gen/genModelTests.py

```diff
@@ -54,8 +59,35 @@
     )
 
     fc_relu_tc(
-        file_name="fc_relu_mse_sgd.info",
+        file_name="fc_relu_mse.info",
         loss_fn_str="mse",
         optimizer=opt.SGD(learning_rate=0.1),
-        debug="initial_input"
+    )
+
+    fc_bn_sigmoid = [
+        K.Input(shape=(3)),
+        K.layers.Dense(10),
+        K.layers.BatchNormalization(),
+        K.layers.Activation("sigmoid"),
+        K.layers.Dense(10),
+        K.layers.Activation("softmax"),
+    ]
+
+    fc_bn_sigmoid_tc = partial(
+        record,
+        model=fc_bn_sigmoid,
+        input_shape=(3, 3),
+        label_shape=(3, 10),
+        optimizer=opt.SGD(learning_rate=1),
+        iteration=10,
+    )
+
+    fc_bn_sigmoid_tc(
+        file_name="fc_bn_sigmoid_cross.info",
+        loss_fn_str="cross_softmax",
+        # debug=["summary", "iteration", "loss"],
+    )
+
+    fc_bn_sigmoid_tc(
+        file_name="fc_bn_sigmoid_mse.info", loss_fn_str="mse",
```

Comment:
can you please elaborate on what does a `.info` file is expected to contain?

Reply:
Done on the doc for the file :)